components/dialog/overview.md

```diff
@@ -45,38 +47,108 @@ The following example demonstrates how to set up the Dialog with its default con
 }
 ````
 
->caption The result from the above code snippet.
+## Predefined Dialogs
+
+Predefined Dialogs are styled substitutes to the standard confirm, alert and prompt dialogs. They are separate from the main Dialog component. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).
+
+## Header
+
+The Dialog provides header customization. [Read more about the Dialog Header]({%slug dialog-header%}).
+
+## Action Buttons
+
+The Dialog provides options for rendering action buttons and customizing their text. [Read more about the Dialog Action Buttons]({%slug dialog-action-buttons%}).
 
-![](images/dialog-basic-configuration.png)
+## Events
 
-## Features
+The Blazor Dialog fires visible change events that you can handle and further customize its behavior. [Read more about the Blazor Dialog events]({%slug dialog-events%}).
 
-The Dialog provides the following features:
+## Dialog Parameters
 
-* `Visible` - `bool` - defines the visibility of the Dialog.
+The Blazor Dialog provides various parameters to configure the component. Also check the [Dialog public API](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Components.TelerikDialog).
 
-* `Title` - `string` - defines the title of the Dialog.
+<style>
+    article style + table {
+        table-layout: auto;
+        word-break: normal;
+    }
+</style>
+| Parameter | Type and Default Value | Description |
+| --- | --- | --- |
+| `ButtonsLayout` | `DialogButtonsLayout` enum <br /> (`Stretched`) | Defines the layout of the actions button in the footer. See more in the [Action Buttons article]({%slug  dialog-action-buttons%})). |
+| `Class` | `string` | Renders a custom CSS class to the `<div class="k-window k-dialog">` element. |
+| `CloseOnOverlayClick` | `bool` | Defines whether clicking on the modal overlay should close the Dialog. |
```

Comment:
"whether" is more correct, but native speakers say the "if" is OK too. And will make the description shorter.

```suggestion
| `CloseOnOverlayClick` | `bool` | Defines if clicking on the modal overlay should close the Dialog. |
```